lib/ui/platform_dispatcher.dart

```diff
@@ -1251,6 +1296,106 @@ class WindowPadding {
   }
 }
 
+/// Area of the display that is obstructed by a hardware feature.
+///
+/// The bounds, which the application can use to guide layout, may be obscured
```

Comment:
Connect the idea of 'bounds' with the input parameter 'rect' (if I am understanding it correctly that rect indicates the bounds referred to here)